src/main/java/edu/unc/lib/boxc/migration/cdm/services/PostMigrationReportService.java

```diff
@@ -99,10 +100,19 @@ public void addWorkRow(String cdmObjectId, String boxcWorkId, int childCount, bo
         String cdmUrl = buildCdmUrl(cdmObjectId, true, isSingleItem);
         String boxcTitle = getParentTitle(cdmObjectId);
         String boxcUrl = this.bxcBaseUrl + boxcWorkId;
+        String matchingValue = null;
         String parentUrl = null;
         String parentTitle = null;
         String objType = ResourceType.Work.name();
-        addRow(cdmObjectId, cdmUrl, objType, boxcUrl, boxcTitle, null, parentUrl, parentTitle, childCount);
+        String sourceFile = null;
+
+        if (isSingleItem) {
+            matchingValue = getMatchingValue(cdmObjectId);
+            sourceFile = getSourceFile(cdmObjectId);
+        }
```

Comment:
I'm wondering if we don't really need this since it looks like single items are also being handled down in addFileRow? We probably don't need the same mapping to show up for both the file and the work